<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myDialog</class>
 <widget class="QDialog" name="myDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnGetHostInfo">
       <property name="text">
        <string>QHostInfo获取本机主机名和IP地址</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnDetail">
       <property name="text">
        <string>QNetworkInterface::allAddess()</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkOnlyIPv4">
       <property name="text">
        <string>只显示IPv4协议地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnALLInterface">
       <property name="text">
        <string>QNetworkInterface::allInterfaces()</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnLookUp">
       <property name="text">
        <string>QHostInfo查找域名和IP地址</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editHost"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnClearText">
       <property name="text">
        <string>清空文本框</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
